tests/conftest.py

```diff
@@ -6,6 +6,12 @@
 from pydantic import BaseModel
 
 
+@pytest.fixture
```

Comment:
since flask 3.0.0 removed _request_ctx_stack so hence the packaged used pytest-flask also removed the associated fixture request_ctx, please see here: https://github.com/pytest-dev/pytest-flask/pull/168. In order to pop the request on the top of the stack, we needed to do via app.test_request_context().